contracts/libs/decimals/DecimalsConverter.sol

```diff
@@ -104,6 +104,41 @@ library DecimalsConverter {
         return convertSafe(amount_, decimals_, round18);
     }
 
+    /**
+     * @notice The function to do the token precision convertion
+     * @param amount_ the amount to convert
+     * @param baseToken_ current token
+     * @param destToken_ desired token
+     * @return the converted number
+     */
+    function roundTokens(
+        uint256 amount_,
+        address baseToken_,
+        address destToken_
+    ) internal view returns (uint256) {
+        return
+            convert(
+                amount_,
+                uint256(IERC20Metadata(baseToken_).decimals()),
+                uint256(IERC20Metadata(destToken_).decimals())
```

Comment:
We can use `decimals` function from the current library

Reply:
sure